release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -146,6 +146,11 @@ For more information, see xref:../updating/updating-cluster-prepare.adoc#updatin
 ==== Minimum required permissions for Microsoft Azure to install and delete an {product-title} cluster
 In {product-title} {product-version}, instead of using the built-in roles, you can now define your custom roles to include the minimum required permissions for Microsoft Azure to install and delete an {product-title} cluster. These permissions are available for installer-provisioned infrastructure and user-provisioned infrastructure.
 
+[id="ocp-4-13-install-configure-vips-to-run-on-control-plane"]
+==== Configuring network components to run on the control plane in vSphere
+
+If you need the virtual IP (VIP) addresses to run on the control plane nodes in a vSphere installation, you must configure the ingressVIP VIP addresses to run exclusively on the control plane nodes. By default, {product-title} allows any node in the worker machine configuration pool to host the ingressVIP VIP addresses. Because vSphere environments deploy worker nodes in separate subnets from the control plane nodes, configuring the ingressVIP virtual IP addresses to run exclusively on the control plane nodes prevents issues from arising due to deploying worker nodes in separate subnets. For additional details, see xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc#configure-network-components-to-run-on-the-control-plane_installing-vsphere-installer-provisioned-network-customizations[Configuring network components to run on the control plane in vSphere].
```

Comment:
One nit: "ingressVIP VIP addresses" is a little awkward and could probably be shortened to "ingressVIP addresses", but otherwise this looks fine.